src/api/ums/umsModule.spec.ts

```diff
@@ -0,0 +1,121 @@
+// tslint:disable:no-string-literal
+import { ReflectiveInjector } from "injection-js";
+import { createMockFor, SpyObj } from "../../../spec/testUtils";
+import { API } from "../../config";
+import { RequestAdapter } from "../../internal/requestAdapter";
+import { deferPromise } from "../../internal/utils";
+import { Client, ClientInit } from "../core/client";
+import { AuthOptions, TokenManager } from "../core/tokenManager";
+import { UMSModule } from "./umsModule";
+
+describe('UMS Module', () => {
+  const projectID = "projectIDTest";
```

Comment:
suggestion for the future: introduce `faker` to avoid creating these literals for testing